Update 500px.py

tom-acco opened the pull request:
Spoke to soon - ran the code a few times to get requests up to see if 500px wigs out. It sure does...

Added in a timer under the followUser - should fix it

tom-acco committed:
Update 500px.py

Issue with followUser timer. Fixed

skuseme commented:
Ran this twice yesterday, run time is significantly longer but requests are spaced nicely.

Just to clarify with the previous merge I had 500px return "Too many requests"

tom-acco committed:
Update README.md

tom-acco committed:
unFollow accepted variable

By setting the variable on line 34 to true, when the acceptedList is cleared of users older than a week, they will also be unfollowed.

tom-acco committed:
Update 500px.py
